sqlserver/datadog_checks/sqlserver/const.py

```diff
@@ -268,3 +268,73 @@
 ]
 
 PROC_CHAR_LIMIT = 500
+
+# Schemas
+DEFAULT_SCHEMAS_COLLECTION_INTERVAL = 1200
```

Comment:
Should this match what we do in postgres? It's once every 10 minutes currently https://github.com/DataDog/integrations-core/blob/master/postgres/datadog_checks/postgres/metadata.py#L27 & there was talk of decreasing it even further

Reply:
I put back 10 mins. Actually, I wanted to talk about it as in my agent-dev kube namespace experiment intake was really slow. Shouldn't we increase it and code that we  send only diffs after the first intake ?  (also for Postgres in the second step) ?
I plan to do another experiment.